python/README.md

```diff
@@ -2,65 +2,68 @@
-  * Imports:
-    - Always at the top of the file, just after any module comments and docstrings and before module globals and constants.
-    - Imports should be grouped from most generic to least generic:
-      - Standard library imports.
-      - Related third party imports.
-      - Local application/library specific imports.
-      - A blank line between each group of imports.
-      - Within each grouping, imports should be sorted lexicographically, ignoring case, according to each module's full package path.
-    - Avoid relative imports, always use the full package name.
+- Maximum line length is 80 characters.
+- Indent code blocks with 4 spaces.
+- Use parentheses sparingly.
+- Naming: `module_name`, `package_name`, `ClassName`, `method_name`, `ExceptionName`, `function_name`, `GLOBAL_CONSTANT_NAME`, `global_var_name`, `instance_var_name`, `function_parameter_name`, `local_var_name`
+  - Avoid dashes (`-`) in any package/module name.
+  - Use single underscore for private/protected methods/variables (avoid "dunder"-s).
+- [Google-style docstrings](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings):
+  - Summary line, blank line, followed by the rest of the docstring.
+  - All sentences must be terminated by a period, question mark, or exclamation point.
+  - The docstring should be descriptive-style ("Fetches rows from a Bigtable.") rather than imperative-style ("Fetch rows from a Bigtable.").
+  - Use `Args:`, `Returns:`, and `Raises:` for documenting functions.
+  - In case of the arguments with a description longer than one line in `Args:`, indent the following lines with 2 spaces.
+- Comments:
+  - Either before the operation or on the same line.
+    - Comments on the same line should start at least 2 spaces away from the code with the comment character #, followed by at least one space before the text of the comment itself.
+  - Comments should be as readable as narrative text, with proper capitalization and punctuation.
+- Imports:
+  - Always at the top of the file, just after any module comments and docstrings and before module globals and constants.
+  - Use `from <module> import <name>` for importing a single object or `import <module>` for importing a module.
+    - Do not use `from <module> import *`.
+  - Imports should be grouped from most generic to least generic:
+    - Standard library imports.
+    - Related third party imports.
+    - Local application/library specific imports.
+    - A blank line between each group of imports.
+    - Within each grouping, imports should be sorted lexicographically, ignoring case, according to each module's full package path.
+  - Avoid relative imports and use the full package name.
+    - The only exception is when importing inside `__init__.py` files. In this case, we use relative imports from the same module. This simplifies the long paths when importing to other modules.
```

Comment:
Please change it back. We have guidelines for formatting lists in Markdown: https://github.com/iai-group/guidelines/tree/main/markdown#lists